a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -57,17 +57,18 @@ def execute(self, listeners, health_mon, vthunder, flavor=None):
                 args.update({'monitor': flavors})
 
         try:
-            health_mon.type = openstack_mappings.hm_type(self.axapi_client, health_mon.type)
+            health_mon_type = openstack_mappings.hm_type(self.axapi_client, health_mon.type)
```

Comment:
Please revert this change. The root cause has been fixed in another PR

Reply:
Reverted